tests/common/db/packaging.py

```diff
@@ -84,7 +82,7 @@ class Meta:
 
     release = factory.SubFactory(ReleaseFactory)
     python_version = "source"
-    filename = factory.fuzzy.FuzzyText(length=12)
+    filename = factory.Faker("file_name")
```

Comment:
Looks like this might not be sufficiently random for some of our tests: https://github.com/pypa/warehouse/runs/4386584039?check_suite_focus=true

Reply:
I think 9d7eb0356cdfc5e4765eeafe82156ed5dcf49f99 will solve it (for now).